### ISSUE TITLE
[DLB]2018-03-31

### ISSUE THREAD

**Issue by projokim** [YBIGTA/data-science-2018#30]
[Batch_Normalization](https://github.com/YBIGTA/DLB/blob/master/Improving%20Neural%20Network/BatchNormalization.ipynb)
